src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.scss

```diff
@@ -190,7 +190,7 @@ $sidebar-width: 200px;
       li a {
         color: vv.$white;
         display: block;
-        font-size: 1.3em;
+        font-size: 1.1em;
```

Comment:
@nSedrickm shouldn't it be 1rem? 1rem = 16px.

Reply:
Thanks @nizamial09. The previous units were em so I just maintained